homeassistant/components/sensor/bunq.py

```diff
@@ -0,0 +1,214 @@
+"""
+Support for balance data via the Bunq Bank API.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.bunq/
+"""
+import asyncio
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_API_KEY
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_point_in_utc_time
+from homeassistant.util import dt as dt_util
+
+REQUIREMENTS = ['bunq-sdk==1.1.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+# minutes between refreshes
+INTERVAL_TO_NEXT_REFRESH = 120
```

Comment:
Rename this to `SCAN_INTERVAL = timedelta(minutes=120)` and Home Assistant will make sure to not call your update function of each sensor once every two hours.

Reply:
Oh I guess you disable polling, that's ok too 👍

Reply:
Indeed, that's whats happening, since in 1 API call you update all accounts (sensors) and that is handled by the BunqData object (stole that mechanism from Buienradar)